### ISSUE TITLE
Swift Version

### ISSUE THREAD

**Issue by rsalesas** [rs/SDAVAssetExportSession#89]
I've converted this to a Swift version. If there's any interest I'm happy to contribute...?

**Comment by aaronwardle** [comment]
That sounds awesome, I still need to get my head around the existing code but a swift version sounds even better, have you found any improvements along the way with the Swift version?

**Comment by rsalesas** [comment]
Yes, but I ended up rewriting, so it's basically completely different. I don't know when I'll get around to trying to post it as a project though.

**Comment by aaronwardle** [comment]
Ah sounds cool, I'm sure people will be interested in it and hopefully help support it I know there are a few issues in the current code, so we have been debating do we write something different too.  We also noticed since iOS12 the encoding seems much quicker now too :D

**Comment by radagrawal** [comment]
@rsalesas  can you share it with me? I am also looking for swift version of SDAVAssetExportSession.

**Comment by Jack-McCabe** [comment]
@rsalesas, are you able to share the swift version?

**Comment by radagrawal** [comment]
I need to it for swift 5. It would be great if you have it.

**Comment by radagrawal** [comment]
> I need to it for swift 5. It would be great if you have it.


I need to it for swift 5. It would be great if you have it.
